### PR TITLE
[New][Author] Add tempusdominus-bootstrap-4 w/ git auto-update via single package.json

### PR DESCRIPTION
**Pull request by Eonasdan**
Pull request for issue: #11453
Related issue(s): # #12337

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/tempusdominus/bootstrap-4
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/tempusdominus-bootstrap-4
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 13
   - Count of stars: 89
   - Count of forks: 28
 * NPM download stats (optional):
   - Downloads in the last day: 63
   - Downloads in the last week: 312
   - Downloads in the last month: 647

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by Eonasdan** [comment]
@extend1994 ok now what? had to make a new per

**Comment by Eonasdan** [comment]
can @dakshshah96 or @extend1994 @sufuf3 @maruilian11 @sashberd help me fix this?

**Comment by sufuf3** [comment]
You merged the master branch into your `tempusdominus-bootstrap-4` branch.
I helped fix the branch and help remove `package-lock.json` which we don't need.
Also, help remove the `version` field, because we don't add this field when we add the lib via single package.json.

I'll review this PR later.

File: ajax/libs/tempusdominus-bootstrap-4/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "url": "https://github.com/tempusdominus/bootstrap-4.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/tempusdominus/bootstrap-4.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "*/"
```
Comment (sufuf3) [inline ajax/libs/tempusdominus-bootstrap-4/package.json]: Let's use `**/*`.
The current pattern can't get the files from git repo.
As extend1994 [said](https://github.com/cdnjs/cdnjs/pull/12337#discussion_r157361881) 
(You can fix this part via GitHub web-site, we can help you squash the commit.)
Thanks.

**Comment by sufuf3** [comment]
@Eonasdan Thanks.
Would you mind revising the pattern from `**/` to `**/*` (add another `*`)
Also fill in the lib's info in first comment?
Many thanks.

**Comment by sufuf3** [comment]
Helped rebase the branch with the latest master and pass CI test.

**Pull request merged**